pulsar-common/src/main/java/org/apache/pulsar/policies/data/loadbalancer/LocalBrokerData.java

```diff
@@ -385,6 +379,7 @@ public Set<String> getBundles() {
         return bundles;
     }
 
+    @VisibleForTesting
```

Comment:
Should we remove this line?

Reply:
Great suggestion.  I have updated the test cases using this method and removed this setter. PTAL @poorbarcode